content/influxdb/v2.0/reference/cli/influx/bucket/create.md

```diff
@@ -83,3 +90,15 @@ influx bucket create \
   --retention 30d \
   --shard-group-duration 2d
 ```
+
+##### Create a bucket with an explicit schema
+
+{{% cloud %}}
+[Explicit bucket schemas](/influxdb/cloud/reference/cli/influx/bucket-schema) are only
```

Comment:
```suggestion
[Explicit bucket schemas](/influxdb/cloud/reference/cli/influx/bucket/create/) are only
```

Reply:
That design issue again....prob would be helpful to also add a link above next to the command `--schema-type` that refers to how to manage bucket schema in case they don't read down into the examples?